chrome/locale/en-US/zotero/preferences.dtd

```diff
@@ -140,7 +140,8 @@
 <!ENTITY zotero.preferences.keys.quicksearch				"Quick Search">
 <!ENTITY zotero.preferences.keys.newItem					"Create a New Item">
 <!ENTITY zotero.preferences.keys.newNote					"Create a New Note">
-<!ENTITY zotero.preferences.keys.toggleRead					"Mark Item Read/Unread">
+<!ENTITY zotero.preferences.keys.toggleAllRead				"Mark All Items As Read/Unread">
+<!ENTITY zotero.preferences.keys.toggleRead					"Mark An Item As Read/Unread">
```

Comment:
This is going to be confusing, since it's not clear that it's about feed items (and some people want read/unread functionality elsewhere, but we're not really interested in offering that). So probably should be "Mark All Feed Items as Read/Unread" and "Mark Feed Item as Read/Unread".